deployments/yaml/manager.yaml

```diff
@@ -1323,7 +1323,7 @@ spec:
         - --leader-elect
         command:
         - /manager
-        image: projects.registry.vmware.com/cnsi/manager:0.3
+        image: projects.registry.vmware.com/cnsi/manager:0.3.2
```

Comment:
Is this generated or hardcoded?

Reply:
hardcoded

Reply:
I think this is supposed to be generated using `cnsi-manager/config/manager/kustomization.yaml` changes.
@4everming can you comment on this? Whether it has to be generated or not?

Reply:
@bgubbala I didn't understand what you meant before, and now I use cnsi-manager/config/manager/kustomization.yaml to change the generated manger version.

Reply:
@qijianshuai  Your latest changes are Good. This is what I was expecting
Also, I have verified your latest changes on my local setup.